benchexec/model.py

```diff
@@ -719,7 +725,14 @@ def after_execution(self, exitcode, forceTimeout=False, termination_reason=None)
             output = []
 
         self.status = self._analyze_result(exitcode, output, isTimeout, termination_reason)
-        self.category = result.get_result_category(self.identifier, self.status, self.properties)
+        if self.multiproperty:
+            for property in self.properties:
+                short_property = property.replace(result._PROP_CALL_SPECIFIC, '')
```

Comment:
This line removes `unreach-call` from property name, therefore tool searches for property by error function name or by name of any other categories ( 'valid-deref', 'valid-free', 'valid-memtrack', 'no-overflow', 'no-deadlock', 'termination'). This is correct for CPAchecker wrapper (if property is not equal to 'valid-deref', 'valid-free', 'valid-memtrack', 'no-overflow', 'no-deadlock', 'termination', it is considered as 'unreach-call'), but I am not sure, if it will be correct for general tool (see my comment in https://github.com/sosy-lab/benchexec/issues/217).